hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/TestCleaner.java

```diff
@@ -1453,8 +1454,7 @@ private void testPendingCompactions(HoodieWriteConfig config, int expNumFilesDel
         .withLogFile(partition, "fileId5", 1, 2)
         .addCommit("011")
         .withBaseFilesInPartition(partition, "fileId7")
-        .withLogFile(partition, "fileId7", 1, 2)
-        .addCommit("013");
```

Comment:
As commented, the earliest instance is 009,  delete commit 013 here.
https://github.com/apache/hudi/blob/ab62ffd77af6bfa92dd111072efd9f684a99bd94/hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/TestCleaner.java#L1171-L1188